libraries/Microsoft.Bot.Builder.Dialogs.Adaptive.Testing/TestActions/AssertReply.cs

```diff
@@ -51,19 +51,20 @@ public override void ValidateReply(Activity activity)
             // if we have a reply
             if (!string.IsNullOrEmpty(this.Text))
             {
+                var description = this.Description != null ? this.Description + "\n" : string.Empty;
                 if (this.Exact)
                 {
                     // Normalize line endings to work on windows and mac
                     if (activity.AsMessageActivity()?.Text.Replace("\r", string.Empty) != this.Text.Replace("\r", string.Empty))
                     {
-                        throw new Exception(this.Description ?? $"Text '{activity.Text}' didn't match expected text: {this.Text}'");
+                        throw new Exception($"${description}Text '{activity.Text}' didn't match expected text: {this.Text}'");
```

Comment:
You don't need the second '$'. .i.e.: Exception($"{description}Text...

Reply:
The thought was that the payloads could be large so didn't want to always do it since exceptions were rare.  Anyway, I updated it to a single string.